docs/configuration.md

```diff
@@ -447,6 +449,45 @@ The possible values are: `gtc` (default), `fok` or `ioc`.
     This is ongoing work. For now, it is supported only for binance and kucoin.
     Please don't change the default value unless you know what you are doing and have researched the impact of using different values for your particular exchange.
 
+### Exchange configuration
+
+Freqtrade is based on [CCXT library](https://github.com/ccxt/ccxt) that supports over 100 cryptocurrency
+exchange markets and trading APIs. The complete up-to-date list can be found in the
+[CCXT repo homepage](https://github.com/ccxt/ccxt/tree/master/python).
+ However, the bot was tested by the development team with only Bittrex, Binance and Kraken,
+ so these are the only officially supported exchanges:
+
+- [Bittrex](https://bittrex.com/): "bittrex"
+- [Binance](https://www.binance.com/): "binance"
+- [Kraken](https://kraken.com/): "kraken"
+
```

Comment:
i've intentionally removed (or rather moved) this section from configuration to exchange.md in 7f4baab420ce98c97deeedb69170c3129828c9b2 - so if there's changes to it, these should be done in that segment too, without readding this segment.